### PR TITLE
Add shellcheck to travis.

### PR DESCRIPTION
**Pull request by dlorenc**
Also fix the current errors.

### CODE REVIEW

**Comment by dlorenc** [comment]
cc @sharifelgamal

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,14 @@
 language: go
 go: 1.7.1 
 install: true
-script: make test
+
+addons:
+  apt:
+    sources:
+    - debian-sid    # Grab shellcheck from the Debian repo (o_O)
```
Comment (sharifelgamal) [inline .travis.yml]: Why sid?

Comment (dlorenc) [inline .travis.yml]: Great question. I copied it from here: https://github.com/koalaman/shellcheck/wiki/TravisCI

Comment (sharifelgamal) [inline .travis.yml]: Ok, as long as it works and that repo stays around, it should be fine.

**Pull request merged**